### DISCUSSION TITLE
Azure Gov - Support

### DISCUSSION THREAD

**Discussion by Johnny32100** [microsoft/PubSec-Info-Assistant#271] [General]
Hello, 

Does Info-Assistant work for Azure Gov subscriptions? I am seeing that Azure Gov doesn't have the option to see OpenAI Service under "Create Resource" 

We have placed our request for OpenAI Service and that was accepted.

**Comment by lon-tierney** [comment]
Great question - this was implemented in our last sprint. A confirmed/documented version should be in Delta release (soon).  If you move to the vNext branch, it should work for you. Deploying to Gov is part of our CI/CD pipeline currently.

**Reply by Johnny32100** [reply]
Wonderful! Thank you! How soon is 'soon' for the document? Would like to share that with my leadership team

**Reply by lon-tierney** [reply]
Targeting this sprint - so ~two weeks if all goes well. Reach out to isat-support@microsoft.com if you need sooner.

**Comment by dayland** [comment]
@Johnny32100, US Gov deployment support is now available in our [0.4-Delta Release](https://github.com/microsoft/PubSec-Info-Assistant/releases/tag/v0.4-Delta). You can find out more about it in our [Deployment Guide](https://github.com/microsoft/PubSec-Info-Assistant/blob/main/docs/deployment/deployment.md) and Optional Feature for [Sovereign Deployment](https://github.com/microsoft/PubSec-Info-Assistant/blob/main/docs/features/features.md#sovereign-region-deployment)
